pkgs/servers/matrix-conduit/default.nix

```diff
@@ -37,8 +37,8 @@ rustPlatform.buildRustPackage rec {
     darwin.apple_sdk.frameworks.Security
   ];
 
-  ROCKSDB_INCLUDE_DIR = "${rocksdb_6_23}/include";
-  ROCKSDB_LIB_DIR = "${rocksdb_6_23}/lib";
+  ROCKSDB_INCLUDE_DIR = "${rocksdb}/include";
+  ROCKSDB_LIB_DIR = "${rocksdb}/lib";
```

Comment:
```suggestion
  env = {
    ROCKSDB_INCLUDE_DIR = "${rocksdb}/include";
    ROCKSDB_LIB_DIR = "${rocksdb}/lib";
  };
```

Reply:
I applied your `env` change but I don't think `buildInputs` is sufficient because the `rocksdb-sys` crate's `build.rs` doesn't seem to use `pkg-config`: https://github.com/rust-rocksdb/rust-rocksdb/blob/v0.21.0/librocksdb-sys/build.rs